scripts/release-notes/index.js

```diff
@@ -1,14 +1,14 @@
-const minimist = require('minimist');
-const { Octokit } = require('@octokit/core');
-const { paginateGraphql } = require('@octokit/plugin-paginate-graphql');
+const minimist = require("minimist");
```

Comment:
The single-quotes are required by our eslint rules.  We need to roll back these changes to double-quotes.  Then, you can run `npm run lint` to check for any other linting issues in this file before pushing a new version. :+1: